db/schema.rb

```diff
@@ -12,38 +10,39 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20_191_115_013_418) do
+ActiveRecord::Schema.define(version: 2019_11_15_013418) do
+
   # These are extensions that must be enabled in order to support this database
-  enable_extension 'plpgsql'
+  enable_extension "plpgsql"
```

Comment:
Seems odd that we'd go from single to double quotes here (and throughout this file)

Reply:
db schema is auto-generated;  it should have been excluded from rubocop to begin with (my bad)